frameworks/PHP/workerman/benchmark_config.json

```diff
@@ -41,35 +39,12 @@
       "webserver": "None",
       "os": "Linux",
       "database_os": "Linux",
-      "display_name": "workerman-postgres",
+      "display_name": "workerman-mysql",
       "notes": "",
       "versus": "php"
     },
-    "async": {
-      "db_url": "/db",
-      "fortune_url": "/fortunes",
-      "port": 8080,
-      "approach": "Realistic",
-      "classification": "Platform",
-      "database": "MySQL",
-      "framework": "workerman",
-      "language": "PHP",
-      "flavor": "PHP7",
-      "orm": "Raw",
-      "platform": "workerman",
-      "webserver": "None",
-      "os": "Linux",
-      "database_os": "Linux",
-      "display_name": "workerman-async-db",
-      "notes": "",
-      "versus": "php",
-      "tags": [
-        "broken"
-      ]
-    },
-    "php8-jit": {
```

Comment:
Please, maintain normal and JIT versions.
That's for test PHP and Workerman it's a really good candidate. And PHP 8.4 come with a complete new JIT.

But the worst thing is that it'll disappear from all historic visualizations.

Reply:
@joanhey 
The without-jit testing has been added. All changes have been locally verified and are working fine.